rockspec/luasql-odbc-2.6.0-3.rockspec

```diff
@@ -0,0 +1,49 @@
+package = "LuaSQL-ODBC"
+version = "2.6.0-3"
+source = {
+  url = "git+https://github.com/keplerproject/luasql.git",
+  branch = "2.6.0",
+}
+description = {
+   summary = "Database connectivity for Lua (ODBC driver)",
+   detailed = [[
+      LuaSQL is a simple interface from Lua to a DBMS. It enables a
+      Lua program to connect to databases, execute arbitrary SQL statements
+      and retrieve results in a row-by-row cursor fashion.
+   ]],
+   license = "MIT/X11",
+   homepage = "http://www.keplerproject.org/luasql/"
```

Comment:
This URL is not valid - there is no `luasql` page on the Kepler project website - I mean - clicking [this](http://www.keplerproject.org/luasql/) won't take you to anything about `luasql`!